### ISSUE TITLE
wemos shield

### ISSUE THREAD

**Issue by DarthWeber** [srg74/ESP32_ethernet#1]
Hi, is it possible to use this with your wemos shield board?

**Comment by srg74** [comment]
Not with shield board, but with latest esp32 dev board.

**Comment by DarthWeber** [comment]
yes, for the new one it is clear. But i have several shield boards already. It is not possible to put the ESP32_ethernet on top of the esp32 ? According to the schematic it looks like the ports are unused...

**Comment by DarthWeber** [comment]
AH, no. There are collisions with the data ports. Sad.
